django_mongodb/query.py

```diff
@@ -139,209 +95,56 @@ def get_cursor(self):
             cursor.limit(int(self.query.high_mark - self.query.low_mark))
         return cursor
 
-    def add_filters(self, filters, query=None):
```

Comment:
I don't see the add_filters method anymore. Is it no longer needed? Was this the function that got originally called on the `WhereNode` instruction?

Reply:
Yes, see django_mongodb/compiler.py old line 139.

Reply:
Yes, it is not longer needed, We have changed it using the as_mql method and it recursive structure. So, add filters was replaced by as_mql method of the WhereNode. This way to compute the filter is more Django like taking advantage of the tree structure.